docs/building-spark.md

```diff
@@ -259,6 +259,12 @@ or
 Java 8 tests are automatically enabled when a Java 8 JDK is detected.
 If you have JDK 8 installed but it is not the system default, you can set JAVA_HOME to point to JDK 8 before running the tests.
 
+## PySpark pip installable
+
+If your are building Spark for use in a Python environment and you wish to pip install it, you will first need to build the Spark JARs as described above. Then you can construct an sdist package suitable for setup.py and pip installable package.
```

Comment:
Sounds like a good idea, I'll add that. Its currently built as part of make-release too but incase someone wants to roll their own patches should cover it.